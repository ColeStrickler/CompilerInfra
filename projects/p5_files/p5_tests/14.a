swag : custom {
    field1 : int;
    field2 : bool;
};

swag2 : custom {
    field1 : bool;
    field2 : bool;
};

g : () -> bool {
    a : swag;
    b : swag2;

    x : bool = a == b;
}
