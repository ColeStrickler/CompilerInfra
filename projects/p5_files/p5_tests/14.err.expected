FATAL [15,16]-[15,17]: Invalid equality operand
Type Analysis Failed
